components/resources/brave_components_resources.grd

```diff
@@ -266,6 +266,7 @@
       <message name="IDS_BRAVE_UI_CLOSE_BALANCE" desc="">Closing Balance</message>
       <message name="IDS_BRAVE_UI_CONTRIBUTE" desc="">Contribute</message>
       <message name="IDS_BRAVE_UI_CONTRIBUTE_ALLOCATION" desc="">Auto-Contribute Allocation</message>
+      <message name="IDS_BRAVE_UI_CONTRIBUTE_TOOLTIP" desc="">Brave Auto-Contribute Settings</message>
```

Comment:
Needs to be just `Auto-Contribute Settings`

Reply:
It was my understanding that auto-contribute references were to be named: `Brave Auto-Contribute` (issue reference https://github.com/brave/brave-core/pull/457)

cc: @jenn-rhim 

If this is the change that needs to be made, should I update UI as well? We shouldn't have differences in locales between ui and here.

Reply:
That was changed yesterday. You can see that we only have Auto-Contribute now (`IDS_BRAVE_UI_CONTRIBUTE_ALLOCATION`)

Reply:
https://github.com/brave/brave-core/pull/502